components/CalculHTTTC.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+import { useState } from 'react';
+import Decimal from 'decimal.js';
+import { Button } from "./ui/Form/Button/Button";
+import { InputWithLabel } from "@/components/ui/Form/InputWithLabel/InputWithLabel";  
+
+const CalculHTTTC = () => {
+  const [montantHT, setMontantHT] = useState<number>(0);
+  const [tauxTVA, setTauxTVA] = useState<number>(20);
+  const [totalTTC, setTotalTTC] = useState<number>(0);
+
+  const handleInputChange = (setter: React.Dispatch<React.SetStateAction<number>>) => (e: React.ChangeEvent<HTMLInputElement>) => {
+    setter(Number(e.target.value));
+  };
+
+  const calculerTTC = () => {
+    const montantHTDecimal = new Decimal(montantHT);
```

Comment:
if you remember i told you to extract all logic from components and create them in separate folder names helpers.js
and create also a test file for this functions. this is not the right place to define those functions.

the project can scale and in that case we needs those functions to be exported and defined separatly from components to reuse them and for readability